wrapper.nim

```diff
@@ -0,0 +1,85 @@
+import os
+import osproc
+import streams
+import strformat
+import dialogs
+
+# Statically includes the installer binary
+const installerBinary = staticRead("/build/libs/fabric-installer-0.7.0.jar")
```

Comment:
staticRead links to the jar or copies it raw into the program?

Reply:
Also the value in the string should probably get set at compile time.

Reply:
It copies it into the exe, and yes, ill prob get gradle to spit out a jar without the version.